src/runtime/node/process/internal/process.ts

```diff
@@ -3,6 +3,7 @@ import type nodeProcess from "node:process";
 import mock from "../../../mock/proxy";
 import empty from "../../../mock/empty";
 import { notImplemented } from "../../../_internal/utils";
+import { ReadStream, WriteStream } from "../../tty/index.mjs";
```

Comment:
Here, we can use `node:tty`. In pure unenv mode, it will use exact same polyfills (via alias) and in hybrid mode, it might leverage runtime (for example workerd) natives.

Reply:
```suggestion
import { ReadStream, WriteStream } from "node:tty";
```